src/Sulu/Bundle/ContentBundle/Command/MaintainResourceLocatorCommand.php

```diff
@@ -93,7 +94,7 @@ private function upgradeByParent(
     }
 
     private function upgradeNode(
-        Structure $page,
+        StructureBridge $page,
```

Comment:
This could/should be `PageBridge`. We could also migrate to use the DM directly, but I don't mind either way, this would be replaced by routing auto hopefully.